app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("NotWearingMaskError")
+        else:
```

Comment:
There is no need for `else` since the code will not execute after the raised exceptions anyway. Please, remove the redundant `else`